pydis_site/apps/api/tests/test_reminders.py

```diff
@@ -107,6 +107,7 @@ def test_delete_known_reminder_returns_204(self):
         response = self.client.delete(url)
 
         self.assertEqual(response.status_code, 204)
+        self.assertRaises(Reminder.DoesNotExist, Reminder.objects.get, **{'id': self.reminder.id})
```

Comment:
As mentioned in another suggestion, I think this can just be a regular kwarg.

```suggestion
        self.assertRaises(Reminder.DoesNotExist, Reminder.objects.get, id=self.reminder.id)
```